DOC-2261 docs(crr): fix Enable the CRR setup (3.10 - 4.1) (#497)
tigergraph/server-docs: modules/cluster-and-ha-management/pages/set-up-crr.adoc

The following setup is needed in order to enable Cross Region Replication.

=== Obtain the latest Backup of the primary data

Retrieve the latest backup, for instance, `pr_latest_backup` from the Primary Cluster. How recent the backup is will determine how much the DR cluster lags behind and how long it will take to catch up with the Primary cluster.

> If there is no proper backup, use `gadmin backup create <pr_latest_backup>` to create one. For more details refer to xref:tigergraph-server:backup-and-restore:backup-cluster.adoc[].

Reviewer: ```suggestion
If there is no suitable backup, use `gadmin backup create <pr_latest_backup>` to create one.
For more details refer to xref:tigergraph-server:backup-and-restore:backup-cluster.adoc[].
```

.Run with the latest backup from its primary cluster.
[source.wrap,console]
----
gadmin backup restore pr_latest_backup --dr
----
+

> Adding the backup tag to `gadmin backup restore <backup tag> --dr` will enable CRR in a regular restore. 

Reviewer: ```suggestion
Adding `--dr` to `gadmin backup restore <backup tag>` enables CRR in a regular restore. 
```

Enabling Cross-Region Replication (CRR) with a backup is always recommended, as the Disaster Recovery (DR) cluster will be restored to the state of the backup. This can help avoid data inconsistency problems caused by unexpected operations or requests before the cluster becomes a DR.

For special cases, such as enabling Cross-Region Replication (CRR) for a new cluster or switching between PR and DR, backup can be omitted.

However, note that these operations should be performed when there is no traffic, because `gadmin backup restore --dr` won't overwrite the data of the cluster before it is set to be a DR.
+
.Instead, simply run:
[source.wrap,console]
----
gadmin backup restore --dr
----

> Specifically, for the switching between PR and DR, before the switch, it is necessary to xref:tigergraph-server:crr:troubleshooting.adoc#_check_data_consistency_between_primary_and_dr[Check data consistency between primary and DR]

Reviewer: ```suggestion
Specifically, when switching between PR and DR, before the switch, it is necessary to xref:tigergraph-server:crr:troubleshooting.adoc#_check_data_consistency_between_primary_and_dr[check data consistency between primary and DR]
```
